### PR TITLE
New Relic login section update

### PR DESCRIPTION
**Pull request by carolyn-shannon**
## Effect
PR includes the following changes:
- Move login section
- update login instructions from SSO to user/password

## Remaining Work
- [x] Waiting for final update to account configuration changes
- [ ] Technical review
- [x] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by carolyn-shannon** [comment]
Looks like this is no longer needed, so archiving.

File: source/content/new-relic.md [hunk 1]
```diff
@@ -27,16 +21,59 @@ Select the **New Relic** tab on your Site Dashboard, and click **Activate New Re
...
 New Relic is automatically enabled for all application containers added to the site, including Multidev environments.
 
+## New Relic Access
+Go to the New Relic workspace on your Dashboard, and click **Go to New Relic**.
+
+### Who is the New Relic account associated with?
+
+* If the site owner is a **user**, the site owner's name and email address is used to create the New Relic account.
+* If the site owner is an **organization**, the name and email address of the organization's Pantheon user who activates New Relic is used to create the New Relic account.
+* Your New Relic **password** is unique to your New Relic account. If you have forgotten or don't yet have a New Relic password, use your Pantheon account email to request a [password reset](https://login.newrelic.com/passwords/forgot).
```
Comment (alexfornuto) [inline source/content/new-relic.md]: @carolyn-shannon this line is new to the doc with this introduction. Not sure if it was in preparation for NR changes, or currently true.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5359 | Difference |
| --- | --- | --- | --- |
| Performance Score | `83` | `78` | `-5` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `93` | `-3` |

Lighthouse Audit Reports:

- [`pr-5359` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17558/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17558/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/new-relic.md (hunk 1 above)
Comment (carolyn-shannon) [reply source/content/new-relic.md]: Good catch. This was based on an earlier conversation but no longer seems to be the case. Can be removed.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5359 | Difference |
| --- | --- | --- | --- |
| Performance Score | `77` | `70` | `-7` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `93` | `-3` |

Lighthouse Audit Reports:

- [`pr-5359` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17568/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17568/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**